src/modules/Snapper.rb

```diff
@@ -604,22 +553,19 @@ def RestoreFiles(snapshot_num, files)
       ret
     end
 
-
     # convert map with userdata to a string
     # { "a" => "1", "b" => "2" } -> "a=1, b=2"
     def userdata_to_string(userdata)
-      return userdata.map { |k, v| "#{k}=#{v}" }.join(", ")
+      userdata.map { |k, v| "#{k}=#{v}" }.join(", ")
     end
 
-
     # convert string with userdata to a map
     # "a=1, b=2" -> { "a" => "1", "b" => "2" }
     def string_to_userdata(string)
-
       tmp1 = string.split(",")
 
       tmp2 = tmp1.map do |s|
-        s.split("=", 2).map { |t| t.strip }
+        s.split("=", 2).map(&:strip)
```

Comment:
you mean `&:` construct or generic idea of this line that removes spaces around splitted elements?